lib/ui/input.tsx

```diff
@@ -10,7 +10,7 @@ const Input = React.forwardRef<HTMLInputElement, InputProps>(
       <input
         type={type}
         className={cn(
-          "flex h-10 w-full rounded-lg border-2 border-solid border-input bg-background px-3 py-2 text-sm ring-offset-background transition-colors file:border-0 file:bg-transparent file:text-sm file:font-medium placeholder:text-muted-foreground/60 hover:border-input-hover focus-visible:border-input-hover focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:cursor-not-allowed disabled:bg-muted disabled:opacity-50",
```

Comment:
I think `focus-visible` and all of this should stay for accessibility purposes. We need this for keyboard navigation. I think there's workarounds to prevent this behavior when using a mouse, but we would need to find a general solution to this.

Reply:
Okay I just reverted this change.